test/cmd/config-changes.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
+trap os::test::junit::reconcile_output EXIT
+
+RESOURCE_DIR="$(dirname "${BASH_SOURCE}")/../resources"
+
+os::test::junit::declare_suite_start "cmd/create"
+
+os::cmd::expect_success 'make build'
+
+os::cmd::expect_success 'oc new-project config'
+
+os::cmd::expect_success 'oc create configmap test-config --from-file=$RESOURCE_DIR/'
```

Comment:
log4j.properties and spark-defaults.conf should probably be put in a subdirectory in RESOURCE_DIR to be clean, the test yaml files in RESOURCE_DIR don't need to be put into the config map

Reply:
changed